src/StreamJsonRpc/PublicAPI.Shipped.txt

```diff
@@ -41,7 +41,7 @@ StreamJsonRpc.JsonRpc.InvokeAsync<TResult>(string targetName, object argument) -
 StreamJsonRpc.JsonRpc.InvokeWithParameterObjectAsync<TResult>(string targetName, object argument = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.Task<TResult>
 StreamJsonRpc.JsonRpc.NotifyWithParameterObjectAsync(string targetName, object argument = null) -> System.Threading.Tasks.Task
 StreamJsonRpc.JsonRpcMethodAttribute
-StreamJsonRpc.JsonRpcMethodAttribute.JsonRpcMethodAttribute(string name) -> void
+StreamJsonRpc.JsonRpcMethodAttribute.JsonRpcMethodAttribute(string name, bool useSingleObjectParameterDeserialization = false) -> void
```

Comment:
This is a breaking API change. Instead of changing the existing method, add an overload that takes both parameters.

Reply:
I pushed a commit to take care of it. But any time you make a change that impacts PublicAPI.Shipped.txt you should carefully consider the breaking API impact.